docker/main/rootfs/usr/local/nginx/conf/nginx.conf

```diff
@@ -149,62 +149,57 @@ http {
 
         location /ws {
             proxy_pass http://mqtt_ws/;
-            proxy_http_version 1.1;
-            proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection "Upgrade";
-            proxy_set_header Host $host;
+            include proxy.conf;
         }
 
         location /live/jsmpeg/ {
             proxy_pass http://jsmpeg/;
-            proxy_http_version 1.1;
-            proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection "Upgrade";
-            proxy_set_header Host $host;
+            include proxy.conf;
         }
 
         location /live/mse/ {
             proxy_pass http://go2rtc/;
-            proxy_http_version 1.1;
-            proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection "Upgrade";
-            proxy_set_header Host $host;
+            include proxy.conf;
         }
 
         location /live/webrtc/ {
             proxy_pass http://go2rtc/;
-            proxy_http_version 1.1;
-            proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection "Upgrade";
-            proxy_set_header Host $host;
+            include proxy.conf;
         }
 
         location ~* /api/go2rtc([/]?.*)$ {
             proxy_pass http://go2rtc;
             rewrite ^/api/go2rtc(.*)$ /api$1 break;
-            proxy_http_version 1.1;
-            proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection "Upgrade";
-            proxy_set_header Host $host;
+            include proxy.conf;
         }
 
         location ~* /api/.*\.(jpg|jpeg|png)$ {
             rewrite ^/api/(.*)$ $1 break;
             proxy_pass http://frigate_api;
-            proxy_pass_request_headers on;
-            proxy_set_header Host $host;
-            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-            proxy_set_header X-Forwarded-Proto $scheme;
+            include proxy.conf;
         }
 
         location /api/ {
             add_header Cache-Control "no-store";
             expires off;
             proxy_pass http://frigate_api/;
-            proxy_pass_request_headers on;
```

Comment:
just wanted to doublecheck, is this not needed (since it's not contained in proxy.conf)?

Reply:
on is the default value, so i dont think its needed